src/reporters/lang/en.js

```diff
@@ -132,6 +132,8 @@ const messages = {
   linkUnregistered: 'Unregistered $0.',
   linkUsing: 'Using linked module for $0.',
 
+  createUnsupportedScope: 'Scoped packages cannot be installed via yarn create for now - read <github issue> to know why.',
```

Comment:
Yeah, maybe we can shell out to the binary specified in package.json of the package as we discussed in the team sync today :)